docs/details/other-components/xml.rst

```diff
@@ -0,0 +1,290 @@
+.. _xml:
+
+====================
+XML - Declarative UI
+====================
+
+Introduction
+------------
+
+LVGL is capable of loading UI elements written in XML.
+Although still under development, the basics are already functional, serving as a preview.
+
+This declarative language serves as the backend for LVGL's UI editor (currently under development),
+which enables faster and more maintainable UI implementation.
+
+Note that, the UI editor is not required to utilize LVGL's XML loading capabilities.
+
+Describing the UI in XML in a declarative manner offers several advantages:
+
+- XML files can be loaded at runtime (e.g., from an SD card) to change the application build.
+- XML is simpler to write than C, enabling people with different skill sets to create LVGL UIs.
+- XML is textual data, making it easy to parse and manipulate with scripts.
+- XML can be used to generate LVGL code in any language.
+- XML helps to separate the view from the logic.
+
+Limitations:
+
+- Only basic widgets are supported with limited functionality.
+- Only a few style properties are supported.
```

Comment:
Should we mention what is supported here? 

Is there a list of what supported somewhere? I suggest to point to it, or mention the supported. features, users that read this section may infer by themselves what are the limitations of a new feature, which can impact the overall experience.

Reply:
Added.

Reply:
Feel free to resolve it .